concent_api/core/message_handlers.py

```diff
@@ -424,25 +434,79 @@ def handle_send_force_subtask_results_response(request, client_message):
     return HttpResponse("", status = 202)
 
 
+def sum_payments(list_of_payments: list = None):
+    assert isinstance(list_of_payments, list)
+
+    return sum([item.amount for item in list_of_payments])
+
+
+def sum_subtask_price(subtask_results_accepted_list: list = None):
+    assert isinstance(subtask_results_accepted_list, list)
+
+    return sum([subtask_results_accepted.task_to_compute.subtask_price for subtask_results_accepted in subtask_results_accepted_list])
+
+
+def sum_amount_price_for_provider(
```

Comment:
1) same here, please use List[X] from typing 
2) please do not make it default to None, if None is not an allowed value

Reply:
Done